rllib/algorithms/appo/appo_catalog.py

```diff
@@ -10,7 +10,7 @@ class APPOCatalog(PPOCatalog):
         - Value Function Head: The head used to compute the value function.
 
     The ActorCriticEncoder is a wrapper around Encoders to produce separate outputs
-    for the policy and value function. See implementations of PPORLModuleBase for
+    for the policy and value function. See implementations of PPORLModule for
```

Comment:
Renamed for consistency:
see also: e.g. AppoLearner as base for AppoTfLearner and AppoTorchLearner.